spec/controllers/queue_items_controller_spec.rb

```diff
@@ -159,4 +159,128 @@
     end
   end
 
+  describe 'PUT :update' do
+
+    let(:user) { user = Fabricate(:user) }
+    let(:queue_items) { queue_items = user.queue_items }
+    let(:first) { first = queue_items.first }
```

Comment:
You can just do `let(:first) { queue_items.first }` - no need to assign that to a local variable `first`.